install/kots/manifests/kots-preflight.yaml

```diff
@@ -84,8 +84,16 @@ spec:
               message: There must be at least one node with the label "gitpod.io/workload_workspace_headless" and value "true"
           - pass:
               message: Node with label exists
+    - customResourceDefinition:
```

Comment:
I moved this up so the memory/CPU checks are in the same section - seemed more logical that way